awx/api/templates/api/_new_in_awx.md

```diff
@@ -10,4 +10,5 @@
 {% if new_in_300 %}> _Added in Ansible Tower 3.0.0_{% endif %}
 {% if new_in_310 %}> _New in Ansible Tower 3.1.0_{% endif %}
 {% if new_in_320 %}> _New in Ansible Tower 3.2.0_{% endif %}
+{% if new_in_330 %}> _New in Ansible Tower 3.3.0_{% endif %}
```

Comment:
A little bit of an aside, but I'm almost of the opinion that we should just drop this structure entirely. I think on a fundamental level this will satisfy this: https://github.com/ansible/awx/issues/73

Reply:
It also sucks that this is something we have to track every release but doesn't give us any other granularity beyond the resources level.

Reply:
Yea, I'm also of the opinion that we should just stop doing this.  `<opinion>`If you want to know what's new in each version of Tower, consult the product release notes.`<opinion>`